tests/unit/backend/test_gcp.py

```diff
@@ -0,0 +1,58 @@
+import os
+
+from pytest import fixture, mark, raises
+from redis_om import NotFoundError
+
+from runner_manager.backend.gcloud import GCPBackend
+from runner_manager.models.backend import Backends, GCPConfig, GCPInstanceConfig
+from runner_manager.models.runner import Runner
+from runner_manager.models.runner_group import RunnerGroup
+
+
+@fixture()
+def gcp_group() -> RunnerGroup:
+    runner_group: RunnerGroup = RunnerGroup(
+        id=2,
+        name="test",
+        organization="test",
+        backend=GCPBackend(
+            name=Backends.gcloud,
+            config=GCPConfig(
+                project_id=os.getenv("GCP_PROJECT_ID"),
+                zone=os.getenv("GCP_ZONE"),
+                service_account_email="default",
+            ),
+            instance_config=GCPInstanceConfig(
+                machine_type="e2-small",
+                image={
+                    "project": "ubuntu-os-cloud",
+                    "family": "ubuntu-2004-lts",
+                },
+                network="default",
+            ),
+        ),
+        labels=[
+            "label",
+        ],
+    )
+    return runner_group
+
+
+@fixture()
+def gcp_runner(runner: Runner, gcp_group: RunnerGroup) -> Runner:
+    # Cleanup and return a runner for testing
+    gcp_group.backend.delete(runner)
+    return runner
+
+
+@mark.skipif(
```

Comment:
Checkout if you can use markers to apply the same logic:
https://docs.pytest.org/en/6.2.x/example/markers.html#marking-platform-specific-tests-with-pytest

Reply:
It's ok, let's leave it as is for now, we'll review once the backend config story is figured out.